plugin/src/main/java/io/jenkins/plugins/casc/CasCGlobalConfig.java

```diff
@@ -15,6 +15,7 @@
 public class CasCGlobalConfig extends GlobalConfiguration {
 
     private String configurationPath;
+    private String mergeStrategy;
```

Comment:
> Instead of CasCGlobalConfig, should it not be SelfConfigurator?

You mean here? Actually, I did use it now. Maybe just remove here. Users can choose the merge strategy from system property. What do you think?

Reply:
The self configurator configures the configuration context
https://github.com/jenkinsci/configuration-as-code-plugin/blob/80e556e277b9a9980659eb1131497758e49bf239/plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationContext.java#L15